deepface/DeepFace.py

```diff
@@ -96,7 +96,7 @@ def verify(
             OpenFace, DeepFace, DeepID, Dlib, ArcFace, SFace and GhostFaceNet (default is VGG-Face).
```

Comment:
you added some facial recognition models for yolo but they are missing here

Reply:
fixed